### PR TITLE
Feat: Select ui-kit

### PR DESCRIPTION
**Pull request by tuul-wq**
- New `Select` component

image

### CODE REVIEW

File: src/renderer/shared/ui-kit/Select/Select.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+}>;
+
+type RootProps = PropsWithChildren<ControlledSelectProps & ContextProps>;
+
+const Root = ({
+  theme = 'light',
+  invalid,
+  disabled,
+  testId = 'Select',
+  open,
```
Comment (sokolova-an) [inline src/renderer/shared/ui-kit/Select/Select.tsx:37]: let's call boolean values for `open` as `isOpen` it's easier to read in components

**Review by sokolova-an — APPROVED** [review]

File: src/renderer/shared/ui-kit/Select/Select.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  parameters: {
+    layout: 'centered',
+  },
+  render: (params) => {
+    const [value, onChange] = useState('');
+
+    return (
+      <Box width="200px">
+        <Select {...params} value={value} onChange={onChange}>
+          <Select.Button placeholder="Select a fruit" />
```
Comment (johnthecat) [inline src/renderer/shared/ui-kit/Select/Select.stories.tsx]: I don't see any use case for separated Button component

File: src/renderer/shared/ui-kit/Select/Select.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  },
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Select>;
+
+export const Default: Story = {};
+
+export const Controlled: Story = {
```
Comment (johnthecat) [inline src/renderer/shared/ui-kit/Select/Select.stories.tsx]: This stories make no sense

File: src/renderer/shared/ui-kit/Select/Select.tsx [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+  onChange: (value: string) => void;
+} & XOR<{
+  open: boolean;
+  onToggle: (value: boolean) => void;
+}>;
+
+type RootProps = PropsWithChildren<ControlledSelectProps & ContextProps>;
+
+const Root = ({
+  theme = 'light',
```
Comment (johnthecat) [inline src/renderer/shared/ui-kit/Select/Select.tsx:33]: Why?

Comment (tuul-wq) [reply src/renderer/shared/ui-kit/Select/Select.tsx:33]: We have a dark colored Select when you scan QR from PV 
image

File: src/renderer/shared/ui-kit/Select/Select.tsx (hunk 1 above)
Comment (tuul-wq) [reply src/renderer/shared/ui-kit/Select/Select.tsx:37]: Select props were based on Dropdown props that has boolean `open` 
Also discussed this moment with @johnthecat and decided to stay with `open`

**Pull request merged**